src/LockFreeIndexSetBuilders.cpp

```diff
@@ -141,8 +141,12 @@ void buildLockFreeBlockIndexset(RAJA::IndexSet& iset,
       }
     }
   } else { /* 3d mesh */
+
```

Comment:
Leave them in for now.  @davidbeckingsale and I plan to get the proxy app repo cleaned up next week. We'll remove them after we've copied them over